src/util/Util.js

```diff
@@ -267,6 +267,22 @@ class Util {
     return { animated: Boolean(m[1]), name: m[2], id: m[3] || null };
   }
 
+  /**
+   * Resolves a partial emoji object from an {@link EmojiIdentifierResolvable}, without checking a Client.
+   * @param {EmojiIdentifierResolvable} emoji Emoji identifier to resolve
+   * @returns {Object} Object with possible `animated`, `name`, and `id` properties
+   * @private
+   */
+  static resolvePartialEmoji(emoji) {
+    if (typeof emoji === 'undefined') return null;
```

Comment:
Is it intended that passing `undefined` returns `null` but passing `null` throws a destructuring error?